candle-examples/examples/custom-ops/cuda_kernels.rs

```diff
@@ -1 +0,0 @@
-pub const LAYERNORM_KERNELS: &str = include_str!(concat!(env!("OUT_DIR"), "/layernorm_kernels.ptx"));
```

Comment:
Intended?

Reply:
Hmm building without cuda will do that it seems.